src/sentry/incidents/logic.py

```diff
@@ -658,7 +658,7 @@ def create_alert_rule(
                 raise TimeoutError
             except ValidationError:
                 alert_rule.delete()
-                raise ValidationError
+                raise
```

Comment:
oops this snuck in from another branch -- the error it fixes here is "not enough parameters" (this would always raise `TypeError` instead of the expected reraise of `ValidationError`